todo_app/data/trello_items.py

```diff
@@ -0,0 +1,103 @@
+import os
+
+import requests
+
+TRELLO_API_HOST = 'https://api.trello.com/1'
+TRELLO_API_KEY = os.getenv('TRELLO_API_KEY')
+TRELLO_API_TOKEN = os.environ.get('TRELLO_API_TOKEN')
+TRELLO_BOARD_ID = os.environ.get('TRELLO_BOARD_ID')
+
+TRELLO_TODO_LIST_ID = os.environ.get('TRELLO_TODO_LIST_ID')
+TRELLO_DONE_LIST_ID = os.environ.get('TRELLO_DONE_LIST_ID')
```

Comment:
This works nicely here, but pre-loading these may bite you in the next module so worth keeping an eye out for it. 

The reason it could be a problem is that these lines of code run when this file is first _imported_. That works perfectly right now, but:
* When we introduce tests we'll sometimes want to do some setup before we start the app
* This can include loading fake/test environment variables
* But these variables will already have had their values assigned by the time we get to our setup, so we won't affect them

We can fix that by messing with our import order carefully, but we typically end up with very fragile code, so a better solution is one of:
* The easiest here, just load the environment variables when they're needed, e.g. in `get_trello_board`:
```python
    query = {
        'key': os.getenv('TRELLO_API_KEY'),
        'token': os.environ.get('TRELLO_API_TOKEN'),
        'cards': 'open',
        'fields': 'id,name,cards',
        'card_fields': 'id,name'
    }
```
* Personally I'd be lazy and go with the above here, but on a real codebase we'd probably prefer a more classic OOP solution; wrap these functions in a class, and load the environment variables in a constructor, e.g. something like:
```python
class TrelloService():

    def __init__(self):
        self.TRELLO_API_KEY = os.getenv('TRELLO_API_KEY')
        ...

    def get_trello_board(self):
        url = "{url}/boards/{id}/lists".format(
             url=self.TRELLO_API_HOST, id=self.TRELLO_BOARD_ID
        )
        ...
```